packages/consensus/src/validation/coinbase.cairo

```diff
@@ -100,19 +124,81 @@ fn validate_coinbase_witness(witness: Span<ByteArray>) -> Result<(), ByteArray>
     Result::Ok(())
 }
 
+
 /// Return BTC reward in SATS
 fn compute_block_reward(block_height: u32) -> u64 {
     shr(5000000000_u64, (block_height / 210000_u32))
 }
 
+/// Calculate wtxid commitment
+fn calculate_wtxid_commitment(wtxid_root: Digest) -> Digest {
+    // construct witness reserved value
+    // 0000000000000000000000000000000000000000000000000000000000000000
+    let mut witness_value_byte: ByteArray = "";
```

Comment:
You can construct it as `"\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0"` (see validate_coinbase_witness)